client/components/card/index.jsx

```diff
@@ -43,7 +44,7 @@ class Card extends Component {
 		const omitProps = [ 'compact', 'highlight', 'tagName' ];
 
 		let linkIndicator;
-		if ( href ) {
+		if ( href || onClick ) {
```

Comment:
We're now omitting `href` and `target` props from the child element even if we have only `onClick` specified, which might be unnecessary. What do you think about moving:

```
else {
	omitProps.push( 'href', 'target' );
}
```

to

```
if ( href ) {
	omitProps.push( 'href', 'target' );
}
```

in order to maintain what we currently have?

Reply:
Needs to be `if ( ! href )` tho 😄

Reply:
Tho it doesn't make sense to omit `href` if there isn't any.

Reply:
🤦‍♂️ 👍

Reply:
Theoretically, `href` could be an empty string, and we wouldn't want to pass it to a `div`, so let's keep that here. Also, removing `target` makes sense too.

Reply:
> Also, removing `target` makes sense too.

Yeah, I had it at that: 6b2f92cdeaf8ac9c3eab0d25fa7e63468ac0c45a

> Theoretically, `href` could be an empty string, and we wouldn't want to pass it to a `div`, so let's keep that here. 

Adding back `href` removal now.